docs/testcases/TC06-003-pass.html

```diff
@@ -136,7 +136,7 @@ <h4 id="TCcode">Code</h4>
   								<h4 id="TCurl">Single-Page URL</h4>
   									<p><a href="../testfiles/TF06/06-003.html">https://section508coordinators.github.io/BaselineTestPages/testfiles/TF06/06-003.html</a></p>
   								<h4 id="TCframe">Embedded Code</h4>
-  									<iframe src="../testfiles/TF06/06-003.html" class="card shadow my-3"></iframe>
+  									<iframe title="embedded test file" src="../testfiles/TF06/06-003.html" class="card shadow my-3"></iframe>
   						<h2 id="TCsteps">Test Steps</h2>
   							<table class="table table-hover">
   								<thead class="thead-dark">
```

Comment:
TC06-003-pass.html:

- Does not present the current baseline format under "Applicable ICT Baseline" as the other Image test cases have